docs/pages/reference/cli/tctl.mdx

```diff
@@ -945,6 +945,26 @@ $ tctl get clusters,users
 
 </Tabs>
 
+## tctl get locks
```

Comment:
Since `locks` is an argument value here, and there's already a section for `tctl get`, I think it would make sense to remove this section.